crm/modules/paypal_payment/paypal_payment.inc.php

```diff
@@ -306,13 +306,13 @@ function paypal_payment_payment_api ($payment, $op) {
 // Table & Page rendering //////////////////////////////////////////////////////
 
 /**
- * Generate payments contacts table
+ * Generate payments contacts table.
  *
  * @param $opts an array of options passed to the paypal_payment_contact_data function
  * @return a table (array) listing the contacts represented by all payments
  *   and their associated paypal email
- */ 
-function paypal_payment_contact_table($opts){
+ */
+function paypal_payment_contact_table ($opts) {
```

Comment:
Spaces around the function parameter list?  I guess I've missed some update in the coding standard?